flexmeasures/data/schemas/reporting/__init__.py

```diff
@@ -0,0 +1,50 @@
+from marshmallow import Schema, fields, validate
+
+from flexmeasures.data.schemas.sensors import SensorIdField
+from flexmeasures.data.schemas.sources import DataSourceIdField
+
+from flexmeasures.data.schemas import AwareDateTimeField, DurationField
+
+
+class TimeBeliefQueryConfigSchema(Schema):
```

Comment:
There is a `d` missing after `Time` (also in the docstring), but you could also consider renaming this to `SearchConfigSchema`, or `BeliefsSearchConfigSchema` (see also comment below).